core/src/main/web/outputdisplay/bko-plot/plotutils.js

```diff
@@ -54,11 +54,24 @@
         222	: "SINGLE_QUOTE"
       };
     return {
+
+      isFirefox: function ()
```

Comment:
these should be application-wide helper functions, not just here.
in fact i am surprised we don't have something like this already, please check.